src/node/pool_manager.rs

```diff
@@ -55,11 +56,9 @@ impl Manager for NodeConnPoolManager {
                 );
 
                 // Take ownership of the `NodeClient` from Pallas
-                let owned = conn.underlying.take().unwrap();
-                // This is the only moment when `underlying` becomes `None`. But
-                // it will never be used again.
-
-                gauge!("cardano_node_connections").decrement(1);
```

Comment:
This PR removed `decrement` for good, which results in wrong value in `GET /metrics`.

Fixed in:
* #180